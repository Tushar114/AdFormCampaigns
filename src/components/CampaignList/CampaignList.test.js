import React from 'react';
import CampaignTable from './Camptable';
import { shallow } from 'enzyme';
import { ReduxProvider } from '../../utility';

describe('Campaign List', () => {
  it('should render Campaign List correctly', () => {
    const component = shallow(
      <ReduxProvider>
        <CampaignTable />
      </ReduxProvider>
    );
    expect(component).toMatchSnapshot();
  });
});
