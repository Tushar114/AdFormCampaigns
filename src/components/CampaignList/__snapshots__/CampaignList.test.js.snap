// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Campaign List should render Campaign List correctly 1`] = `ShallowWrapper {}`;
